flatdata-rs/lib/src/vector.rs

```diff
@@ -349,7 +349,7 @@ mod tests {
 
         assert_eq!(v[0].x(), 10..20);
         assert_eq!(v[1].x(), 20..30);
-        assert_eq!(v[2].x(), 30..0);
+        assert!(v[2].x().is_empty());
```

Comment:
See previous comment, this does not test the same

Reply:
Done.